src/components/tables/Striped.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+const Striped = ({ columns, data }) => (
+    <table className="f6 w-100 mw8 center" cellSpacing="0">
+        <thead>
+            <tr className="stripe-dark">{
+                columns.map((text, i) =>
+                    <th className="fw6 tl pa3 bg-white" key={i}>
+                        { text }
+                    </th>
+                )
+            }</tr>
+        </thead>
+        <tbody className="lh-copy">{
+            data.map((arr, i) =>
+                <tr className="stripe-dark" key={i}>{
+                    arr.map((text, j) =>
+                        <td className="pa3" key={j}>{text}</td>
+                    )
+                }</tr>
+            )
+        }</tbody>
+    </table>
+)
+
+Striped.defaultProps = {
+    columns: ['Name', 'Username', 'Email', 'ID'],
+    data: [
+        ['Hassan Johnson', '@hassan', 'hassan@companywithalongdomain.co', '14419232532474'],
```

Comment:
How come these aren't objects instead of arrays? Still approving the PR, but just wanted to know why.